### PR TITLE
Logging implementation.

### PR DESCRIPTION
**Pull request by AlexeyUlyashev**

### CODE REVIEW

File: serv_logger.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Creates logger object, which report formatted logs on console and to file.
+"""
+
+
+import logging
+import sys
+
+
+# pylint: disable = invalid-name
```
Comment (zharkovav) [inline serv_logger.py]: Pls create function (which will return logger object) and move all staff there.

File: serv_logger.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# pylint: disable = redefined-variable-type
+log = logging.getLogger('main')
+log.setLevel(logging.DEBUG)
```
Comment (zharkovav) [inline serv_logger.py]: Log level should be configurable

File: server.py [hunk 3]
```diff
@@ -161,6 +164,8 @@ def registration(self):
             try:
                 self.check_registration_data(nick, password)
             except RegistrationError as err:
+                log.info('User entered not correct registration '
+                         'data: %s, %s.', nick, password)
```
Comment (zharkovav) [inline server.py]: Do not write password to log

File: serv_logger.py (hunk 1 above)
Comment (AlexeyUlyashev) [reply serv_logger.py]: Chenges

File: serv_logger.py (hunk 2 above)
Comment (AlexeyUlyashev) [reply serv_logger.py]: Implemented.

**Pull request closed without merging**